.vscode/settings.json

```diff
@@ -19,5 +19,6 @@
     "python.testing.unittestEnabled": false,
     "python.testing.pytestEnabled": true,
     "python.linting.mypyCategorySeverity.error": "Warning",
-    "python.linting.mypyEnabled": true
+    "python.linting.mypyEnabled": true,
+    "jupyter.notebookFileRoot": "${workspaceFolder}"
```

Comment:
Why do we need jupyter settings? This entry could be moved into the user settings.